book/src/gobject_memory_management.md

```diff
@@ -92,9 +91,7 @@ Yes! That is exactly what the [`Rc`](https://doc.rust-lang.org/std/rc/struct.Rc.
 The `Rc` counts the number of strong references created via `Clone::clone` and released via `Drop::drop`, and only deallocates it when this number drops to zero.
 We call every object containing a strong reference a shared owner of the value.
 If we want to modify the content of our [`Rc`](https://doc.rust-lang.org/std/rc/struct.Rc.html),
-we can use the [`RefCell`](https://doc.rust-lang.org/std/cell/struct.RefCell.html) type.
-`RefCell` then checks Rust's borrow rules during run time, namely that there can only be one mutable borrow at a time or multiple immutable borrows.
-`RefCell` panics if these rules are violated.
```

Comment:
I think this is not a good idea. `Cell` is much more limited and at least I rarely can use it. It would be useful to at least mention the constraints here (must be `Copy`) and the alternative (`RefCell`).

Reply:
I will revert this change